pytorch_lightning/callbacks/hpu_stats_monitor.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2021 Habana Labs, Ltd. an Intel Company
+# All Rights Reserved.
+#
+# Unauthorized copying of this file or any element(s) within it, via any medium
+# is strictly prohibited.
+# This file contains Habana Labs, Ltd. proprietary and confidential information
+# and is subject to the confidentiality and license agreements under which it
+# was provided.
+#
+
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+hpu Stats Monitor
+=================
+
+Monitor and logs hpu stats during training.
+
+"""
+
+import torch
+
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks.base import Callback
+
+
+class HPUStatsMonitor(Callback):
+    """Automatically monitors and logs hpu stats during training stage.
+
+    Args:
+        save_dir: directory to save the logs.
+        exp_name: name of the experiment.
+    """
+
+    def __init__(self, log_save_dir: str = "habana_ptl_logs", exp_name: str = "default"):
+        super().__init__()
+        self.log_save_dir = log_save_dir
+        self.exp_name = exp_name
+
+    def on_init_end(self, trainer: "pl.Trainer") -> None:
+        from pytorch_lightning import loggers as pl_logger
+
+        self.tb_logger = pl_logger.TensorBoardLogger(save_dir=self.log_save_dir, name=self.exp_name)
+        trainer.logger = self.tb_logger
```

Comment:
1. `on_init_end` is deprecated from the callback api
2. the trainer already has loggers attached it. the callback should not override the existing loggers set on the trainer. just use the loggers already available from `trainer.loggers`

Reply:
There has been previous comments too on hpu_stats_monitor. I'll remove it from this PR and bring it back with a subsequent PR with the addition of more features